code/src/main/kotlin/com/expediagroup/sdk/v2/core/extension/client/HttpRequestExtension.kt

```diff
@@ -0,0 +1,19 @@
+package com.expediagroup.sdk.v2.core.extension.client
+
+import com.google.api.client.http.HttpRequest
+import com.google.api.client.http.InputStreamContent
+import okio.Buffer
+
+fun HttpRequest.getContentBuffer(resetContent: Boolean = true) : Buffer =
+    Buffer().let { buffer ->
+        synchronized(this) {
+            content.writeTo(buffer.outputStream())
```

Comment:
This function might throw `IOException`, correct? we might need to add `@Throws`?

Reply:
Done.